package/gluon-core/luasrc/lib/gluon/upgrade/140-firewall-rules

```diff
@@ -81,8 +81,17 @@ end
 
 
 -- We can't put mesh_wan into this zone, as mesh_wan is the same
--- interface as wan, which has its own zone
-uci:set('firewall', 'wired_mesh', 'network', {'mesh_lan'})
+-- interface as wan, which has its own zone but allow any other defined
+-- gluon_wired interface addicted to bat0 to communicate
+mil = {}
+uci:foreach('network', 'interface',
+	function(i)
+		if i['proto'] == 'gluon_wired' and i['.name'] ~= 'mesh_wan' then
```

Comment:
This could use a comment explaining that `mesh_wan` uses the same underlying interface as `wan` and is therefore not assigned to a firewall zone itself.

Reply:
The comment is above the block and remained from the previous version. Maybe we could move it directly to the line where the actual filtering happens

Reply:
Sounds good